test/unit/helpers/xhr/MFA_REQUIRED_allFactors_autoPushTrue.js

```diff
@@ -0,0 +1,279 @@
+define({
```

Comment:
no need to create this file. We should do some function in the tests where we change the json we have in ```MFA_REQUIRED_allFactors.js```, similar to other tests like [this one](https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/EnrollOnPrem_spec.js#L66-L72)